CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 1.7.0
+
+- POTENTIALLY BREAKING Bug Fix: [Validate variable usage in according to spec](https://github.com/absinthe-graphql/absinthe/pull/1141). This could break incoming documents previously considered valid. Skip the `Absinthe.Phase.Document.Arguments.VariableTypesMatch` phase to avoid this check. See `Absinthe.Pipeline` on adjusting the document pipeline.
```

Comment:
@XiXiaPdx this seems like it could potentially impact us, have we assessed the impact of this breaking change? Let's talk on Slack.